peekingduck/cli.py

```diff
@@ -291,16 +295,16 @@ def _create_nodes_from_config_file(
     config_path: str, project_dir: Path, node_type_choices: click.Choice
 ) -> None:
     """Creates custom nodes declared in the config file."""
-    run_config_path = Path(config_path)
-    if not run_config_path.is_absolute():
-        run_config_path = (project_dir / run_config_path).resolve()
-    if not run_config_path.exists():
+    pipeline_path = Path(config_path)
```

Comment:
does the function argument/CLI option have to be updated to `pipeline_path` as well?

Reply:
Similar `pipeline` vs `configuration` vs `pipeline configuration` discussion.
The nomenclature should be clear once we sort out the philosophy and terminology.